nix/status-go/mobile/default.nix

```diff
@@ -20,7 +20,9 @@
   ios = callPackage ./build.nix {
     platform = "ios";
     platformVersion = "8.0";
-    targets = [ "ios" "iossimulator" ];
+    targets = let
+      targetsString = lib.getEnvWithDefault "IOS_STATUS_GO_TARGETS" "ios;iossimulator";
+    in lib.splitString ";" targetsString;
```

Comment:
I think the env variable is fine, but I wouldn't use it here. I think we should follow the same pattern as above and have `abis` in arguments like for `android`, and use `getEnvWithDefault "IOS_STATUS_GO_TARGETS" "ios;iossimulator"` in `nix/mobile/ios/shells/status-go.nix`.

What do you think?